src/energyplus/ForwardTranslator.cpp

```diff
@@ -555,6 +555,13 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     }
     translateAndMapModelObject(*runPeriod);
 
+    // ensure that output table summary reports exists
+    boost::optional<model::OutputTableSummaryReports> summaryReports = model.getOptionalUniqueModelObject<model::OutputTableSummaryReports>();
+    if (!summaryReports){
+      summaryReports = model.getUniqueModelObject<model::OutputTableSummaryReports>();
```

Comment:
To maintain backward compat, either `OutputTableSummaryReports` needs to have it's Ctor directly call `addSummaryReport("AllSummary")` or you need to do it here by adding a line `summaryReports.addSummaryReport("AllSummary");`

Reply:
I now see you actually do that in the FT `translateOutputTableSumaryReports` method, but I like that less, see comments over there too.